### PR TITLE
Implement felt to bigint functions

### PR DESCRIPTION
**Pull request by pefontana**
# Implement felt to bigint functions
Since various crates have to work with BigInts and BigUints operations, here we implement some Felt to BigInt and Biguint conversions.
* pub fn felt_to_biguint(felt: Felt) -> BigUint
* pub fn felt_to_bigint(felt: Felt) -> BigInt
* pub fn biguint_to_felt(biguint: &BigUint) -> Felt
* pub fn bigint_to_felt(bigint: &BigInt) -> Felt

## Does this introduce a breaking change?

No

### CODE REVIEW

**Review by LucasLvy — CHANGES_REQUESTED** [review]

**Comment by Oppen** [comment]
Since we now have the types in the same crate, would it make sense to implement the `From<Felt> for BigUint` and the like?

**Comment by LucasLvy** [comment]
Maybe something like `From<Felt> for BigUint && BigInt` and a function implemented on the felt struct `to_biguint` idk, i don't really like this kind of helper functions, or maybe in a utils crate

File: crates/starknet-types-core/src/felt.rs [hunk 1]
```diff
@@ -346,6 +356,19 @@ impl Felt {
...
+            .to_le_digits()
+            .into_iter()
+            .flat_map(|limb| [limb as u32, (limb >> 32) as u32])
+            .collect();
+        BigUint::new(big_digits)
+    }
+
+    pub fn to_bigint(&self) -> BigInt {
+        self.to_biguint().to_bigint().unwrap()
+    }
```
[the remark this answers is not in the document]
Comment (pefontana) [reply crates/starknet-types-core/src/felt.rs]: Yes, I thought about doing something like that, but I actually think that the `to_bigint() `and `to_biguint() `methods are more explicit and readable

**Comment by pefontana** [comment]
> Maybe something like `From<Felt> for BigUint && BigInt` and a function implemented on the felt struct `to_biguint` idk, i don't really like this kind of helper functions, or maybe in a utils crate

Sure @LucasLvy !
Done!

**Review by LucasLvy — APPROVED** [review]

**Pull request merged**